dev/com.ibm.ws.logging.osgi/resources/com/ibm/ws/logging/internal/osgi/resources/OSGiMessages.nlsprops

```diff
@@ -30,15 +30,19 @@
 # -------------------------------------------------------------------------------------------------
 
 # OSGI Log Message
-OSGI_MSG001={0} {1} {2} {3} {4}
+OSGI_MSG001=CWWKE0703I: {0} {1} {2} {3}
```

Comment:
This is a global comment: What are the variables for {0} {1} {2} {3} {4}? Are they complete sentences, and is it enough information for a user to determine an action, as the useraction section of the message suggests?

Reply:
I do not really like the approach here, but I also didn't have a good alternative suggestion so just kept with the current design.  These variables are only used as place holders to file in different components from an OSGi LogEntry.  The reason the nlsprops is used here is to provide the message code so a user can look it up in the documentation for the explanation and user action.

As to how good the information is that the user gets, that all depends on what another bundle logs to the OSGi LogService.  We cannot really control that here in this code.